File: Cosmos-IOS/Cosmostation/Cosmostation/Model/KavaClaimMultiplier.swift
//
//  KavaClaimMultiplier.swift
//  Cosmostation
//
//  Created by 정용주 on 2020/10/08.
//  Copyright © 2020 wannabit. All rights reserved.
//

import Foundation

public class KavaClaimMultiplier {
    var name: String = ""
    var months_lockup: String = ""
    var factor: String = ""
    
    init() {}
    
    init(_ dictionary: [String: Any]) {
        self.name = dictionary["name"] as? String ?? ""
        self.months_lockup = dictionary["months_lockup"] as? String ?? ""
        self.factor = dictionary["factor"] as? String ?? ""
    }
}

File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/model/KavaClaimMultiplier.java
package wannabit.io.cosmostaion.model;

import com.google.gson.annotations.SerializedName;

public class KavaClaimMultiplier {

    @SerializedName("name")
    public String name;

    @SerializedName("months_lockup")
    public String months_lockup;

    @SerializedName("factor")
    public String factor;
}
